dydx_withdrawal/serializer/withdrawal_serializer.py

```diff
@@ -17,10 +17,6 @@ class TransferSerializer(serializers.Serializer):
 
 class FastWithdrawalSerializer(serializers.Serializer):
     position_id = serializers.IntegerField(required=True)
-    credit_asset = serializers.CharField(required=True)
-    credit_amount = serializers.CharField(required=True)
-    debit_amount = serializers.CharField(required=True)
+    withdrawalamount = serializers.CharField(required=True)
     to_address = serializers.CharField(required=True)
-    lp_position_id = serializers.CharField(required=True)
-    expiration_epoch_seconds = serializers.CharField(required=True)
-    signature = serializers.CharField(required=True)
+
```

Comment:
why did we remove all these lines ?

Reply:
We don't need this line

Reply:
Because we don't need take all of them fr.req body i have written in functions for them